docs/features/techdocs/configuration.md

```diff
@@ -0,0 +1,72 @@
+---
+id: configuration
+title: TechDocs Configuration Options
+description:
+  Reference documentation for configuring TechDocs using app-config.yaml
+---
+
+Using the `app-config.yaml` in the Backstage app, you can configure TechDocs
+using several options. This page serves as a reference to all the available
+configuration options for TechDocs.
+
+```yaml
+# File: app-config.yaml
+
+techdocs:
+
+  # TechDocs makes API calls to techdocs-backend using this URL. e.g. get docs of an entity, get metadata, etc.
+
+  requestUrl: http://localhost:7000/api/techdocs
```

Comment:
Not to be addressed as part of this PR, but noting that we are in general using the backend [discovery mechanism](https://github.com/backstage/backstage/blob/master/packages/backend/src/plugins/scaffolder.ts#L45) for resolving the backend plugin paths.

Reply:
Had similar thoughts when deploying TechDocs on demo.backstage.io website. Will create an issue around it to deprecate the use of `requestUrl` and `storageUrl` in configs.